### ISSUE TITLE
[Anon] Bug report: Dark_Eldar_Codex_(2014).catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-7th-edition#1816]
**File:** Dark_Eldar_Codex_(2014).catz

**BattleScribe version:** 1.15.07

**Platform:** Windows

**Dropbox:** No

**Description:** The Forgeworld Model "Dark Eldar Raven Fighter" is not listed in the combined arms detachment.  There is a "show FW models" option at the top with checkboxes for "HQ" and "Heavy Support", although the heavy support choices are shown, there are no HQ models from forgeworld. I don't know if this is a bug with the type of unit, but the "Raven Fighter" from ForgeWorld is missing from the list.

If anyone could fix this I would be very greatful, just picked up the model! Thanks

**Comment by OftKilted** [comment]
The Dark Eldar Raven Strike Fighter is listed as a 'Fast Attack' choice once you select all of the 'Show Forgeworld' options.

**Comment by OftKilted** [comment]
The Dark Eldar Raven Strike Fighter has been moved to the 'Fast Attack' slot in the most recent IA:Aeronautica, and shows up in the Fast Attack slot when Forgeworld is selected for 'Fast Attack'
Closing issue.

**Comment by Norion12** [comment]
No, the option in fast attack is for the Razorwing Jetfighter. This is a different model. I have included a screenshot.
![de-bug](https://cloud.githubusercontent.com/assets/17184366/12988351/f153c608-d0c5-11e5-9eb7-5a0b4038fcdd.png)

**Comment by Norion12** [comment]
the name of the forgeworld unit is "Dark Eldar Raven Fighter", the Razorwing Jetfighter is a different GW Model.  Also the check boxes do not include Fast Attack, just HQ and Heavy.

**Comment by OftKilted** [comment]
Add the Forgeworld for Fast Attack. That option is not selected on your screenshot.

**Comment by Norion12** [comment]
There is no checkbox for fast attack, only HQ and heavy

**Comment by OftKilted** [comment]
Reopening to investigate issue with Fast Attack Selection problem.

**Comment by Norion12** [comment]
Thanks :) Also I think the checkbox for HQ might be an issue, there are no Forgeworld HQ models to show.

**Comment by hisop** [comment]
Someone has jsut copied over the hide by section code from SM remove all the check boxes and just have a global "show" or do what was recommended to me for IK and forgo the hiding completely and just append [FW] at the end like the Tau. the main reason C:SM is hidden is as the file is to massive otherwise for initial loading.

**Comment by Norion12** [comment]
Its still able to be repoduced right now, the model cant be selected.

**Comment by Norion12** [comment]
This is still considered a bug right? Im not able to select the unit

**Comment by OftKilted** [comment]
Clear out your data, and redownload. This will allow you to verify functionality. It is possible that the version number was not updated after prior work done, I'll have to see if I can replicate the issue. We will take a look at validating the 'Show/Hide' functionality for the catalog.

**Comment by Norion12** [comment]
I deleted the data repository and downloaded the new one. I installed via file. The issue still exists.

**Comment by Norion12** [comment]
I used to do some coding, I looked at the source code and these are the issues:

line 209 - Hidden="true" // this hides check box for Show FW models Elites
line 217 - Hidden="true" // this hides check box for Show FW models Fast Attack

These should both be marked "false" (HQ and Heavy are marked false, and these checkboxes show up)

The rules for the raven fighter are in the code, but you cant display them with the checkbox for Fast Attack missing.

Hope this helps. I appreciate all the work you guys do!
